libraries/chain/hotstuff/hotstuff.cpp

```diff
@@ -84,21 +77,20 @@ bool pending_quorum_certificate::add_strong_vote(const std::vector<uint8_t>& pro
    assert(index < _num_finalizers);
    if (!_strong_votes.add_vote(proposal_digest, index, pubkey, sig))
       return false;
-   size_t weak   = num_weak();
-   size_t strong = num_strong();
+   _strong_accumulated_weight += _finalizer_weights[index];
 
    switch (_state) {
    case state_t::unrestricted:
    case state_t::restricted:
-      if (strong >= _quorum) {
```

Comment:
I really prefer the short names. Could we keep `_strong` and `_weak`?

Reply:
Those are class members and used across the file. Longer names help readability. `_strong` and `_weak` also imply state.

Reply:
> Those are class members and used across the file. Longer names help readability. `_strong` and `_weak` also imply state.

Can we call them `_strong_sum` and `weak_sum`?